notary-integration-test/src/integration-test/kotlin/integration/btc/BtcNotaryIntegrationTest.kt

```diff
@@ -85,22 +85,24 @@ class BtcNotaryIntegrationTest {
      */
     @Test
     fun testDeposit() {
-        val randomName = String.getRandomString(9)
-        val testClient = "$randomName@$CLIENT_DOMAIN"
-        val btcAddress = integrationHelper.registerBtcAddress(notaryConfig.bitcoin.walletPath, randomName)
-        val initialBalance = integrationHelper.getIrohaAccountBalance(
-            testClient,
-            btcAsset
-        )
-        val btcAmount = 1
-        integrationHelper.sendBtc(btcAddress, btcAmount)
-        Thread.sleep(20_000)
-        val newBalance = integrationHelper.getIrohaAccountBalance(testClient, btcAsset)
-        assertEquals(
-            BigDecimal(initialBalance).add(integrationHelper.btcToSat(btcAmount).toBigDecimal()).toString(),
-            newBalance
-        )
-        assertTrue(btcNotaryInitialization.isWatchedAddress(btcAddress))
+        assertTimeoutPreemptively(timeoutDuration) {
```

Comment:
I don't think we need `assertTimeoutPreemptively` in every test. Can we use this construction in tests known to be long-runing only?